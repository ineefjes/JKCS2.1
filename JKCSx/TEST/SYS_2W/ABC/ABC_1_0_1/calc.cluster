3
/wrk/kubeckaj/DONOTREMOVE/Apps/JKCS2.0/JKCSx/../TOOLS/STRUCTURES/ABC/oh.xyz 1
/wrk/kubeckaj/DONOTREMOVE/Apps/JKCS2.0/JKCSx/../TOOLS/STRUCTURES/JACOB/aq.xyz 0
/wrk/kubeckaj/DONOTREMOVE/Apps/JKCS2.0/JKCSx/../TOOLS/STRUCTURES/JACOB/aqp.xyz 1
* 9
