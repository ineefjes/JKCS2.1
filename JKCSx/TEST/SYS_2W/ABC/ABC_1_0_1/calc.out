                                                      
       * * * * * * * * * * * * * * * * * *            
      *       ____   _____ _           _   *          
     *  /\   |  _ \ / ____| |         | |  * * * * *  
    *  /  \  | |_) | |    | |_   _ ___| |_ ___ _ __  *
   *  / /\ \ |  _ <| |    | | | | / __| __/ _ \ '__| *
  *  / ____ \| |_) | |____| | |_| \__ \ ||  __/ |  *  
  * /_/    \_\____/ \_____|_|\__,_|___/\__\___|_|  *  
  *                                                *  
  * * * * * * * * * * * * * * * * * * * * * * * * *   
                                                      

ABCluster 2.0
Specific flag:  official version
Platform:       LINUX/UNIX
Compiled by:    junzhang@gellmann.scs.illinois.edu
Compiling date: Jul 30 2018 10:38:43
C++ compiler:   g++
C++ options:    -O3 -Os -fno-exceptions --std=c++0x -fopenmp -fPIC -msse4 

Contributors:
Jun Zhang(University of Cologne): Design and coding.

For any suggestions, comments, and questions on the theory and
implementation of ABCluster, please feel free to contact:

Dr. Jun Zhang, ZhangJunQcc@gmail.com

If you used this program in your work, please cite this reference:

[1] Zhang, J.; Dolg, M.; ABCluster: The Artificial Bee Colony Algorithm for
    Cluster Global Optimization. Phys. Chem. Chem. Phys. 2015, 17, 24173-24181.
[2] Zhang, J.; Dolg, M.; Global Optimization of Clusters of Rigid Molecules by the
    Artificial Bee Colony Algorithm. Phys. Chem. Chem. Phys. 2016, 18, 3003-3010.

Program starts at Fri Sep  6 13:25:19 2019.
Command line: " /wrk/kubeckaj/DONOTREMOVE/Apps/ABCluster-2.0-Linux/rigidmol-optimizer calc.inp "
Process ID:   59297
Run by:       kubeckaj@taito-login4.csc.fi
Available CPU cores:    16
Read "OMP_NUM_THREADS": Unset

Random number testing (integers): 121333064...2106265760
Random number testing (fractals): 0.44694459...0.18108210

 -- Input/Output Parameters --
Input file name:      calc.inp
Cluster file name:    calc.cluster
Result file name:     calc.(cluster/xyz/gjf)
Temporary file name:  abclusterkubeckajtaito-login4.csc.fi59297-OPT
# of saved LMs:       600
LMs are stored in:    calc-LM

 -- ABC Control Parameters --
Population size:                   10
Maximal generation:                10
Scout limit:                        4
Amplitude:                 9.00000000

 -- Cluster Components: from [ calc.cluster ] --
 -----------------------------------------------------------
   Source                                       #   Charge
 -----------------------------------------------------------
   /wrk/kubeckaj/DONOTREMOVE/Apps/JKCS2.0/JKCSx/../TOOLS/STRUCTURES/ABC/oh.xyz     1   -1.000
   /wrk/kubeckaj/DONOTREMOVE/Apps/JKCS2.0/JKCSx/../TOOLS/STRUCTURES/JACOB/aq.xyz     0    0.000
   /wrk/kubeckaj/DONOTREMOVE/Apps/JKCS2.0/JKCSx/../TOOLS/STRUCTURES/JACOB/aqp.xyz     1    1.000
 -----------------------------------------------------------
   Dimension: 0, an isolate cluster.
 -----------------------------------------------------------
   External electric field:           0.00000000 Volt/Angstrom
 -----------------------------------------------------------

Artificial Bee Colony Optimization Starts!
----------------------------------------------------------------------------------
 * Population: 10; Maximal cycles: 10
 * EM = Employed Bee; Trigonometric mutation
 * OL = Onlooker Bee; ABC/best/2(0.5)+ABC/current/2(0.5);
        Tournament selection from 5 nectars.
 * SC = Scout Bee; Random nectar; Limit = 4
----------------------------------------------------------------------------------
    #       Diversity            maxE            minE    byEM    byOL    bySC
----------------------------------------------------------------------------------
    1      0.07572037   -732.60201619   -732.79754763       6       3       0
    2      0.07286742   -732.62001247   -732.79754763       3       1       0
    3      0.07135419   -732.64928491   -732.79754763       2       3       0
    4      0.04420570   -732.64928491   -732.79754763       3       0       0
    5      0.00539685   -732.77955795   -732.79754763       2       0       0
    6      0.00000035   -732.79754761   -732.79754763       1       0       0
    7      0.05865938   -732.60201619   -732.79754763       0       0       1
    8      0.09255297   -732.60201619   -732.79754737       1       0       4
    9      0.08120897   -732.60201619   -732.79754763       3       0       5
   10      0.07945935   -732.60201619   -732.79754732       3       0       0
----------------------------------------------------------------------------------
Artificial Bee Colony Optimization Finished!

 -- Results Report --
 * 7 LMs will be saved in [ calc-LM ].
      #          Energy
      0   -732.79754737
      1   -732.79724439
      2   -732.77955795
      3   -732.64928318
      4   -732.63129018
      5   -732.62001247
      6   -732.60201619

 * Final Global Minimal Energy:   -732.79754763
 * The Global Minimum is saved as: [ calc-OPT.(cluster/xyz/gjf) ]

===== (^ _ ^) =====
Normal termination at Fri Sep  6 13:25:19 2019.
Total run time: 0 seconds = 0 days 0 hours 0 minutes 0 seconds.
